.github/workflows/integration.yml

```diff
@@ -25,4 +25,4 @@ jobs:
           pip install -r requirements.txt
 
       - name: Run tests
-        run: python -m unittest main_test.py --verbose
+        run: python run_tests.py
```

Comment:
The reason not all tests were executed is because there's a specific test file in the command. I don' have strong preference over test structure or using loader tho.

Reply:
Yeah, I wasn't even aware there were other test files when I originally did this. Was just taking what was mentioned in the README as truth. This update now runs all tests that match the pattern in `/tests` directory.